tests/utils/constants/constants.go

```diff
@@ -33,6 +33,9 @@ var (
 	WorkSpace           = "WORKSPACE"
 	RHCSPrefix          = "rhcs"
 	TFYAMLProfile       = "tf_cluster_profile.yml"
+	DefaultMPLabels     = map[string]string{
```

Comment:
Let's make it into a config file like backend.

Reply:
can you send a reference to that in the backend repo? could not find how it was implemented

Reply:
Let's keep it for current PR. I will send you via slack about configuration in backend.

Reply:
ack, tnx!